cityofaustin/atd-moped
# Removes unused `moped_proj_phases` columns

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10880

## Testing
**URL to test:**  Local

**Steps to test:**

1. Start the cluster + editor. Verify that the projects list loads as expected.
2. Add and remove a project phase to a project, filling out all fields.
3. Navigate to the Dashboard tab, verify that project loads with out issue.
4. Finally, go to the signal dashboard and verify it loads without issue: `https://localhost:3000/moped/views/signal-projects`

---
#### Ship list
